src/test/java/seedu/address/logic/commands/AddStudentCommandTest.java

```diff
@@ -258,16 +258,6 @@ public boolean hasModule(Module module) {
         public void addModule(Module module) {
             throw new AssertionError("This method should not be called.");
         }
-
-        @Override
-        public void deleteModule(Module module) {
-            throw new AssertionError("This method should not be called.");
-        }
-
-        @Override
-        public Module getModule(Module module) {
-            throw new AssertionError("This method should not be called.");
-        }
```

Comment:
Err did you have to delete this?

Reply:
eh during conflict resolve, there was duplicate, hence i just removed the duplicate

Reply:
Hmmm just took a look this and realised its another issue. Sorry about this.

Reply:
Yeah same, I think these should not be deleted?